[Label] Basic Labels had different height and adjusted (attached) image labels

lubber-de opened the pull request:
## Description
- `basic labels` have a border which increases the height in comparison to usual labels.
When inline together the line height gets increased
- Because of that, a `basic image label` was not properly supported
- All attached labels were not really attached to the boundary of a segment (as every other `attached` component is doing)  . That was especially visible when a `basic attached label` was used

## Testcase
### Before
Watch the height and width difference on basic labels as well as the misaligned image labels and unattached borders in attached labels
https://jsfiddle.net/lubber/wt0heq5k/60/

### After
https://jsfiddle.net/lubber/wt0heq5k/99/

## Screenshots
### Before
Watch the height and width difference on basic labels as well as the misaligned image labels and unattached borders in attached labels
![image](https://user-images.githubusercontent.com/18379884/83977589-e4635d00-a901-11ea-93b0-1e727d5afc88.png)
![image](https://user-images.githubusercontent.com/18379884/83982318-36b67500-a926-11ea-8912-911ba13832ca.png)
![image](https://user-images.githubusercontent.com/18379884/83982354-7d0bd400-a926-11ea-9f8e-701b393af601.png)


### After
![image](https://user-images.githubusercontent.com/18379884/83982306-1d152d80-a926-11ea-8a7d-635caed81bf0.png)
![image](https://user-images.githubusercontent.com/18379884/83982332-4e8df900-a926-11ea-8f48-d2f23c73eaab.png)
![image](https://user-images.githubusercontent.com/18379884/84023157-dfe18780-a987-11ea-9791-e74b9f288a5b.png)
![image](https://user-images.githubusercontent.com/18379884/84023174-e839c280-a987-11ea-97c3-69adcca734a1.png)



## Closes
#1430

prudho commented:
Probably a little bit off topic, but the last image seems to be rounded at position `bottom left`, which seem wrong.

lubber-de commented:
>  but the last image seems to be rounded at position `bottom left`, which seem wrong.

@prudho  You've got Eagle Eyes 😆 
Fixed by https://github.com/fomantic/Fomantic-UI/pull/1507/commits/1922bf7197ea87e472fe6a6f96a54d4562215e8a now. Changed screenshot and updated jsfiddle accordingly
![image](https://user-images.githubusercontent.com/18379884/84023174-e839c280-a987-11ea-97c3-69adcca734a1.png)
@exoego Please review again
